examples/CMakeLists.txt

```diff
@@ -1,38 +1,37 @@
-project(examples)
-cmake_minimum_required(VERSION 2.8.10)
-
 # The examples need a few additional dependencies (e.g. boost filesystem and OpenCV highgui):
-
-#check installed version in order to include the correct OpenCV libraries
-#version variable is defined from project root's CMakeLists
-if("${OpenCV_VERSION_MAJOR}$" EQUAL 2)
-  message(STATUS "OpenCV 2.x detected")
-  find_package(OpenCV 2.4.3 REQUIRED core imgproc highgui)
-elseif("${OpenCV_VERSION_MAJOR}$" EQUAL 3)
-  message(STATUS "OpenCV 3.x detected - including imgcodecs for compatibility")
-  find_package(OpenCV 3.0.0 REQUIRED core imgproc imgcodecs)
-endif()
-message(STATUS "OpenCV include dir found at ${OpenCV_INCLUDE_DIRS}")
-message(STATUS "OpenCV lib dir found at ${OpenCV_LIB_DIR}")
-# This allows us to compile in RelWithDebInfo. It'll use the Release-version of OpenCV:
-set_target_properties(${OpenCV_LIBS} PROPERTIES MAP_IMPORTED_CONFIG_RELWITHDEBINFO RELEASE)
+find_package(OpenCV REQUIRED core highgui)
 
 hunter_add_package(Boost COMPONENTS system filesystem program_options)
-find_package(Boost CONFIG REQUIRED system filesystem program_options)
+find_package(Boost 1.50.0 COMPONENTS system filesystem program_options REQUIRED)
+message(STATUS "Boost found at ${Boost_INCLUDE_DIRS}")
 
-if(MSVC)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj")
-endif()
+# Simple model fitting (orthographic camera & shape to landmarks) example:
+add_executable(fit-model-simple fit-model-simple.cpp)
+target_link_libraries(fit-model-simple eos ${OpenCV_LIBS} ${Boost_LIBRARIES})
+target_link_libraries(fit-model-simple "$<$<CXX_COMPILER_ID:GNU>:-pthread>$<$<CXX_COMPILER_ID:Clang>:-pthreads>")
```

Comment:
We can use `Threads::Threads` here, see:
* https://cmake.org/cmake/help/latest/module/FindThreads.html